pretrain/openwebtext_trainer.py

```diff
@@ -60,11 +60,17 @@ def configure_model(self) -> None:
         self.module.apply(self.module._init_weights)
 
     def configure_optimizers(self) -> torch.optim.Optimizer:
+        if self.module is None:
```

Comment:
Shouldn't it be a decorator, since the code is exactly the same in all the methods below?

Reply:
I'm following the KISS engineering principle here

Reply:
Ok. Fair enough.